schemas/concrete/MPTitle.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$id": "MPTitle.json",
```

Comment:
Can we call this "MPTitles" instead? since it includes multiple titles.

Reply:
done